content/applications/essentials/activities.rst

```diff
@@ -3,236 +3,332 @@ Activities
+     currently installed in the database.
 
-If :guilabel:`Trigger Next Activity` is selected in the :guilabel:`Chaining Type` field, the field
-beneath changes to: :guilabel:`Trigger`. From the :guilabel:`Trigger` field drop-down menu, select
-the activity that should be launched once this activity is completed.
+- :guilabel:`Folder`: this field **only** appears if :guilabel:`Upload Document` is selected for the
+  :guilabel:`Action`.
 
-In the :guilabel:`Schedule` field, choose a default deadline for these activities. To do so,
-configure a desired number of :guilabel:`Days`, :guilabel:`Weeks`, or :guilabel:`Months`. Then,
-decide if it should occur :guilabel:`after completion date` or :guilabel:`after previous activity
-deadline`.
+   Using the drop-down menu, select the :guilabel:`Folder` the document is saved to.
+- :guilabel:`Default User`: select a user from the drop-down menu to automatically assign this
+   activity to the selected user when this activity type is scheduled. If this field is left blank,
+   the activity is assigned to the user who creates the activity.
+- :guilabel:`Default Summary`: enter a note to include whenever this activity type is created.
 
-This :guilabel:`Schedule` field information can be altered before the activity is scheduled.
+  .. note::
+     The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields are
+     included when an activity is created. However, they can be altered before the activity is
+     scheduled or saved.
 
-When all configurations are complete, click :guilabel:`Save`.
+- :guilabel:`Keep Done`: tick this box to keep activities marked as `Done` visible in the
+  :ref:`activity view <activities/activity>`.
+- :guilabel:`Default Note`: enter any notes to appear with the activity.
 
-.. image:: activities/triggered-activities.png
-   :alt: Schedule new activity popup with emphasis on Done and launch next button.
+Next Activity section
+~~~~~~~~~~~~~~~~~~~~~
 
-.. note::
-   When an activity has the :guilabel:`Chaining Type` set to :guilabel:`Trigger Next Activity`,
-   marking the activity as `Done` immediately launches the next activity listed in the
-   :guilabel:`Trigger` field.
+It is possible to have another activity either suggested or triggered. To do so, configure the
+:guilabel:`Next Activity` section.
+
+- :guilabel:`Chaining Type`: select either :guilabel:`Suggest Next Activity` or  :guilabel:`Trigger
+  Next Activity` from the drop-down menu.
+
+  This does **not** appear if :guilabel:`Upload Document` is selected for the :guilabel:`Action`.
+- :guilabel:`Suggest/Trigger`: depending on what is selected for the :guilabel:`Chaining Type`, this
+  field either displays :guilabel:`Suggest` or :guilabel:`Trigger`. Using the drop-down menu, select
+  the activity to recommend or schedule as follow-up tasks to the activity type.
+- :guilabel:`Schedule`: configure when the next activity is suggested or triggered. First, enter a
```

Comment:
This is a very helpful field! I think it's a little hard to conceptualize generally, and I suggest keeping the first sentence:
```suggestion
- :guilabel:`Schedule`: configure when the next activity is suggested or triggered.
```
Then, put the rest of the paragraph you wrote for this field in an admonition block :smile:, something like:

.. example::
   To schedule a follow-up call three days after the first call, do this...

Reply:
I prefer to keep the generic altogether so the steps aren't sepaarated. What I did was space itout, so visually it wasn't a huge chunk of text, so I added a few paragraph returns (each step has it's own paragraph now).